src/interfaces/resource-interfaces/sample-form-interface.ts

```diff
@@ -0,0 +1,8 @@
+/** Interface to define the resource sample form. */
+
+export interface SampleFormIF {
+    header?: string
+    text?: string
+    downloadDocLabel?: string
+    downloadDocPath?: string
+}
```

Comment:
This seems pretty specific (or rather, not global to any old sample form). Can you come up with a more specific name?

Reply:
Sorry, I meant the interface name should be specific to its use -- it's not a generic sample form, is it?

Please revert the property name changes.

Reply:
Ok, i'll fix this up